### PR TITLE
Sprint Review Changes

### PR DESCRIPTION
**Pull request by PamudiAbeykoon**

### CODE REVIEW

**Review by JinapriyaM — APPROVED** [review]

File: front-end/src/App.tsx [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+import { SignUp } from './containers/SignUpPage/SignUpPage';
+import { Login } from './containers/LoginPage/LoginPage';
+import { Admin } from './containers/AdminPage/AdminPage';
+
+function App() {
+  return (
+    <div className="App">
+      <Router>
+        <Routes>
+          <Route path="/main" element={<Main />} />
```
Comment (shamal-iroshan) [inline front-end/src/App.tsx:13]: can store these paths in a constant file

File: front-end/src/api/authApi.tsx [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+  password: String,
+  role: String,
+) => {
+  try {
+    const response = await apiInstance.post('/register', {
+      username,
+      password,
+      role,
+    });
+    console.log(response.data.message);
```
Comment (shamal-iroshan) [inline front-end/src/api/authApi.tsx:18]: remove unwanted logs

File: front-end/src/components/NavigationBar/NavigationBar.tsx [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+  const handleAddUsers = () => {
+    navigate('/admin');
+  };
+  const handleViewUserTable = () => {
+    navigate('/main');
+  };
+
+  const handleLogOutClick = async () => {
+    dispatch(logoutUser())
+    localStorage.removeItem('token');
```
Comment (shamal-iroshan) [inline front-end/src/components/NavigationBar/NavigationBar.tsx:22]: don't store tokens in local storage

File: front-end/src/redux/saga.tsx [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+    const createdStudent = yield call(createStudentApi, newStudent);
+    //success action
+    yield put(createStudentSuccess(createdStudent));
+  } catch (error) {
+    console.error('Error creating new student in postgresql', error);
+  }
+}
+
+export default function* studentSaga() {
+  yield takeLatest(getAllStudents.type, getAllStudentsSaga);
```
Comment (shamal-iroshan) [inline front-end/src/redux/saga.tsx:68]: can directly listen to the action, no need to get the type

**Review by shamal-iroshan — CHANGES_REQUESTED** [review]
fix commented things

**Review by shamal-iroshan — APPROVED** [review]

**Pull request merged**